src/Raven.Server/ServerWide/Commands/CompareExchangeCommands.cs

```diff
@@ -22,7 +22,7 @@ public abstract class CompareExchangeCommandBase : CommandBase
 
         public static string GetActualKey(string database, string key)
         {
-            return ("db/" + database + "/" + key).ToLowerInvariant();
+            return (database + "/" + key).ToLowerInvariant();
```

Comment:
isn't this breaking backward-compatibility?

Reply:
The previous implementation, broke it. This should fix it :)